addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -133,6 +133,27 @@ $i: 1;
         }
     }
 
+    we-button.o_we_mobile {
+
+        & svg {
+            fill: $o-we-color-danger;
+            width: 12px;
+            height: 100%;
+
+            &:hover {
+                fill: lighten($o-we-color-danger, 10%);
+            }
+        }
+
+        &.active svg {
+            fill: $o-we-sidebar-content-field-color;
+
+            &:hover {
+                fill: lighten($o-we-sidebar-content-field-color, 10%);
+            }
+        }
+    }
```

Comment:
There is already code which sets the default color. Even if it is nice to avoid `:not` in general, here it means not duplicating rules so just define rules for `:not(.active)` case ;)

Reply:
```scss
    we-button.o_we_mobile {
        & svg {
            width: 12px;
            height: 100%;
            fill: $o-we-color-danger;

            &:hover {
                filter: brightness(110%);
            }
        }

        &.active svg {
            fill: $o-we-sidebar-content-field-color;
        }
    }
```
I don't know where I should use `:not`?

Reply:
The part with `$o-we-sidebar-content-field-color` should not be necessary. Only the red color is a custom behavior of the option.

Reply:
I think it's necessary because of `.active` doesn't not change the fill color property.

Reply:
Ok, if indeed the active class changes the color it's needed :+1: